pages/vi/profiles/andrewyosh.md

```diff
@@ -0,0 +1,30 @@
+# Andrew Yoshimura
+
+### About Me:
+
+> Hello, I am Andrew and I am currently seeking a bachelor's degree in computer science at the University of Louisiana at Lafayette.  I enjoy programming, music and video games.  My favorite programming language is c# :p
+
+![alt text](https://www.ullafayettefoundation.org/image/home-page/Louisiana-Wall-and-Martin-Hall.jpg)
```

Comment:
Note "alt text" essentially become `alt` attribute in HTML. It need to be something descriptive in some cases when you work on real projects.

Reply:
Oh okay. Thanks.